transcom/mymove
# B 21490 INT - Storybook/Happo Fixes

## [B-21490](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21490)

## Summary

Fixes unintended happo changes/failures found in [the main PR for this backlog](https://github.com/transcom/mymove/pull/14265).

The failures related to the Document Reviewer appear to be unrelated to the changes for the BL but I fixed them before considering that.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Verify you can still switch to your secondary GBLOC
2. Run `make storybook`
3. Verify GblocSwitcher.stories.jsx, TableQueue.stories.jsx, and DocumentViewer.stories.jsx build correctly.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
